main.cpp

```diff
@@ -94,33 +89,48 @@ struct IntType
     int* intTypeHeap = nullptr;
 }; 
 
+//FloatType definition ///////////////////////////////////////////
 struct FloatType
 {
     FloatType(float);
     ~FloatType();
     operator float() const { return *floatTypeHeap; }
-    FloatType& add(float rhs);
-    FloatType& subtract(float rhs);
-    FloatType& multiply(float rhs);
-    FloatType& divide(float rhs);
+
+    FloatType& operator+=( const float other);
+    FloatType& operator-=(const float other);
+    FloatType& operator/=( const float other);
+    FloatType& operator*=(const float other);
+
+    FloatType& operator=(const FloatType& other); 
```

Comment:
the Rule of 3/5/0 hasn't been covered yet.  Your `main()` isn't doing any assignment operations so you have no need for this.
- [ ] Remove this `operator=` function.